SYCL/GroupMask/Basic.cpp

```diff
@@ -0,0 +1,86 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// REQUIRES: gpu
+// UNSUPPORTED: cuda, hip
+// GroupNonUniformBallot capability is supported on Intel GPU only
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+//==---------- Basic.cpp - SYCL Group Mask basic test ----------*- C++ -*---==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
```

Comment:
The test doesn't cover all APIs added by the extension. Do I understand correct that there will be more tests that close this gap?

Reply:
full API is covered in llvm/intel in-tree tests. Here covering group_ballot API and some basic tests.

Reply:
I'm trying to say that it would be nice to have a test which verifies that this macro is really defined to catch cases if some patch breaks it.

Reply:
I have added test for the extension macro to intel/llvm: https://github.com/intel/llvm/pull/4481/commits/db1f5b822ee05308eb6edbf687ef903eeda13d12
It can be later extended with other supported extensions.

Reply:
Thanks.